manifest/manifest.mjs

```diff
@@ -5,11 +5,14 @@ import * as utils from "../utilities/shared.mjs"
 export async function findTheManifestByID(id=null){
    let manifest = null
    if(!utils.validateProjectID(id)) return manifest
+
+   // Mock a pause for endpoints that fail, to mock the time it takes for some async stuff to decide it failed.
    const mockPause = new Promise((resolve, reject) => {
      setTimeout(() => {
        resolve(null)
      }, 3500)
    })
+
    if(id){
       if(id && id===7085) {
          manifest = await fetch("https://t-pen.org/TPEN/manifest/7085").then(resp => resp.json()).catch(err => {
```

Comment:
You can just return the fetch. You don't need to ever set `manifest`.

Reply:
instead of all these nulls, you can return your mockPause async null simulator.

Reply:
This function does not return a promise.  It returns a JSON object or null.